charts/osm/templates/_helpers.tpl

```diff
@@ -2,3 +2,9 @@
 {{- define "osm.namespace" -}} 
 {{ default .Release.Namespace .Values.OpenServiceMesh.osmNamespace}} 
 {{- end -}}
+
+{{/* Default tracing address */}}
+{{- define "osm.tracingAddress" -}}
+{{- $address := printf "jaeger.%s.svc.cluster.local" (include "osm.namespace" .) -}}
+{{ default $address .Values.OpenServiceMesh.tracing.address}} 
```

Comment:
We don't need a default if we are always going to use `jaegar.<osm-namespace>.svc.cluster.local`. We could just remove the default from `values.yaml`?